projects/js-packages/connection/README.md

```diff
@@ -53,7 +53,6 @@ The component displays the connection button and handles the connection process,
 - *apiRoot* - string (required), API root URL.
 - *apiNonce* - string (required), API Nonce.
 - *registrationNonce* - string (required), registration nonce.
-- *onRegistered* - callback, to be called upon registration success.
```

Comment:
I removed because:

1. we were not using it
2. I'm not sure how to capture the end of the action from within the connectButton component. How do I add a callback to run after the action is done? The only way I see is by changing another state property, as we do with `userIsConnecting`
3. When the registration is done, we redirect the user. So we might have race conditions if we attach anything there...

Reply:
I gave this some extra thought and I agree that while it was nice as an option for our consumers it only makes sense to keep it for cases without redirect involved, eg site-only connections.
So for now we can let it go indeed.